src/arrayMethodSort.js

```diff
@@ -2,11 +2,24 @@
 
 /**
  * Implement method Sort
- */
+//  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction =
+  (a, b) => String(a) > String(b)) {
+    for (let a = this.length; a >= 0; a--) {
+      for (let i = 0; i < this.length - 1; i++) {
+        const value1 = this[i];
+        const value2 = this[i + 1];
+
+        if (compareFunction(value1, value2) > 0) {
+          this[i] = value2;
+          this[i + 1] = value1;
```

Comment:
```suggestion
          [this[i + 1], this[i]] = [this[i], this[i + 1]]
```
Here is small tip on how to swap elements using a restructuring assignment

Reply:
Done